pkg/rpm.bzl

```diff
@@ -158,7 +166,6 @@ pkg_rpm = rule(
         "debug": attr.bool(default = False),
 
         # Implicit dependencies.
-        "rpmbuild_path": attr.string(),
```

Comment:
This should be documented as an incompatible change when opportune.

Reply:
Also, this will break the rpm tests in ``//tests/rpm`.

With this change, we should be able to put the RPM tests back in CI.  At the very least, CentOS CI images should have `rpmbuild` in them now, as of bazelbuild/continuous-integration#1039

Reply:
The point of this CL was mostly to see if I can get correct target skipping working.  On macos I now see
```
bazel test ... -- -//experimental/tests/rpm:test_rpm_manifest
//distro:packaging_test                                         (cached) PASSED in 21.4s
//experimental/tests:pfg_bad_section_test                       (cached) PASSED in 0.1s
...
//tests:make_rpm_test                                           (cached) PASSED in 0.7s
//tests:path_test                                               (cached) PASSED in 0.6s
//tests:pkg_deb_test                                            (cached) PASSED in 0.7s
//tests:pkg_tar_test                                            (cached) PASSED in 0.8s
//tests:zip_test                                                (cached) PASSED in 0.8s
@experimental_test_external_repo//pkg:pfg_local_file_in_extrepo (cached) PASSED in 0.1s
//experimental/tests/rpm:pkg_rpm_basic_test                             SKIPPED
//experimental/tests/rpm:pkg_rpm_smoke                                  SKIPPED

Executed 0 out of 44 tests: 42 tests pass and 2 were skipped.
```

So all the things that require rpmbuild get gracefully skipped.

Reply:
I put back rpmbuild_path. You can use it instead of the toolchain, but you will get a warning.